### ISSUE TITLE
display two help when asking help of a command

### ISSUE THREAD

**Issue by EtienneM** [Scalingo/cli#382]
`scalingo help create` displays both help of the command and global help.

It's the same for help of any commands

```
$ scalingo help create
NAME:
   scalingo create - Create a new app

USAGE:
   scalingo create [command options] [arguments...]

CATEGORY:
   Global

DESCRIPTION:
   Create a new app:
   Example:
     'scalingo create mynewapp'
     'scalingo create mynewapp --remote "staging"'

OPTIONS:
   --remote value     Remote to add to your current git repository (default: "scalingo")
   --buildpack value  URL to a custom buildpack that Scalingo should use to build your application                                                                                           

NAME:
   Scalingo Client - Manage your apps and containers

USAGE:
   scalingo [global options] command [command options] [arguments...]

VERSION:
   1.9.0

AUTHOR:
   Scalingo Team <hello@scalingo.com>

COMMANDS:
     help  Shows a list of commands or help for one command
[...]
```

**Comment by johnsudaar** [comment]
I'm unable to reproduce it with the current master version. Is this still an issue @EtienneM ?

**Comment by EtienneM** [comment]
Fixed in c7e0191
